src/components/ProviderLogin.svelte

```diff
@@ -0,0 +1,139 @@
+  onMount(async () => {
+    // This must be in onMount because the extension requires that you have a window to attach to.
+    // Since this project is precompiled, there will be no window until onMount
+    const polkadotExt = await import('@polkadot/extension-dapp');
+    thisWeb3Enable = polkadotExt.web3Enable;
+    thisWeb3Accounts = polkadotExt.web3Accounts;
+  });
+
+  selectedNetwork.subscribe(async (network) => {
+    console.log('selectedNetwork changed to ' + network);
+    if (network != undefined && network != Network.NONE) {
+      let info = networkToInfo[network];
+      console.dir(info);
+      let endpoint = networkToInfo[network].endpoint;
+      console.log('endpoint: ' + endpoint);
+      try {
+        await getApi(endpoint?.toString() ?? '', thisDotApi, wsProvider);
+        await fetchAccounts(network, thisWeb3Enable, thisWeb3Accounts, thisDotApi.api as ApiPromise);
+        await updateConnectionStatus(thisDotApi.api as ApiPromise);
+      } catch (e) {
+        console.log(e);
+        alert(
+          `could not connect to ${
+            endpoint?.toString() || 'empty value'
+          }. Please enter a valid and reachable Websocket URL.`
+        );
+      }
+    }
+  });
+
+  storeProviderAccounts.subscribe((accounts) => {
+    console.log('storeProviderAccounts changed');
+    console.dir(accounts);
+    providerAccounts = Object.fromEntries(Object.entries(accounts).map(([key]) => [key, key]));
+  });
+
+  function connect() {
+    $isLoggedIn = true;
+    pageContent.dashboard();
+  }
+
+  function becomeProvider() {
+    pageContent.becomeProvider();
+  }
+
+  function networkChanged() {
+    console.log('networkChanged');
+    console.log('selectedNetworkAsString: ' + selectedNetworkAsString);
+    $selectedNetwork = selectedNetworkAsString as Network;
+  }
+
+  function accountChanged() {
+    console.log('accountChanged');
+    console.log('selectedAccountAsString: ' + selectedAccountAsString);
+    transactionSigningAddress.set(selectedAccountAsString);
+  }
+</script>
+
+<div id="provider-login" class="content-block w-single-block flex flex-col gap-4">
+  <BlockSection title="Provider Login">
+    <DropDownMenu
+      id="network"
+      label="Select a Network"
+      bind:action={selectedNetworkAsString}
+      onChange={networkChanged}
+      placeholder=""
+      options={networks}
+    ></DropDownMenu>
+    <input
+      type="text"
+      id="other-endpoint-url"
+      placeholder="wss://some.frequency.node"
+      bind:value={customNetwork}
+      disabled={$selectedNetwork != Network.CUSTOM}
+      class:hidden={$selectedNetwork != Network.CUSTOM}
+    />
+    <DropDownMenu
+      id="controlkeys"
+      label="Select a Provider Control Key"
+      bind:action={selectedAccountAsString}
+      onChange={accountChanged}
+      placeholder=""
+      options={providerAccounts}
+    ></DropDownMenu>
+    <Button id="connect-button" title="Connect" disabled={!canConnect} action={connect}></Button>
```

Comment:
Same nit. Also, would recommend deleting the title and just passing the value as a child to match the same conventions as a normal button. Slightly more flexible if we want to add an icon or something too. Up to you.